skaha/src/main/java/org/opencadc/skaha/SkahaAction.java

```diff
@@ -180,40 +182,21 @@ protected void initRequest() throws Exception {
         LocalAuthority localAuthority = new LocalAuthority();
         URI gmsSearchURI = localAuthority.getServiceURI("ivo://ivoa.net/std/GMS#search-0.1");
 
-        // get all the user's groups
-        if (!CredUtil.checkCredentials()) {
-            throw new IllegalStateException("cannot access delegated credentials");
-        }
-        GMSClient gmsClient = new GMSClient(gmsSearchURI);
-        List<Group> memberships = gmsClient.getMemberships(Role.MEMBER);
-
-        final Group skahaUsersGroupObj;
-        try {
-            skahaUsersGroupObj = new Group(new GroupURI(URI.create(skahaUsersGroup)));
-        } catch (Exception e) {
-            throw new RuntimeException(e);
-        }
-
-        if (!memberships.contains(skahaUsersGroupObj)) {
+        log.debug("using iam to fetch group details");
+        IvoaGroupClient ivoaGroupClient = new IvoaGroupClient();
+        GroupURI skahaUsersGroupUri = new GroupURI(URI.create(skahaUsersGroup));
+        Set<GroupURI> skahaUsersGroupUriSet = ivoaGroupClient.getMemberships(gmsSearchURI);
+        log.debug("skahaUsersGroupUriSet is " + skahaUsersGroupUriSet);
+        if (!skahaUsersGroupUriSet.contains(skahaUsersGroupUri)) {
+            log.info("user is not a member of skaha user group ");
```

Comment:
In openCADC software, the INFO level of log is supposed to only be used by our RestAction framework.  So, could you please change the log.info lines to be log.debug instead?  We normally run services in INFO level so don't want too much noise in there.

Reply:
I have changed that specific log to debug level